pytorch_lightning/metrics/classification.py

```diff
@@ -146,13 +148,10 @@ class PrecisionRecallCurve(TensorCollectionMetric):
         >>> pred = torch.tensor([0, 1, 2, 3])
         >>> target = torch.tensor([0, 1, 2, 2])
         >>> metric = PrecisionRecallCurve()
-        >>> prec, recall, thr = metric(pred, target)
-        >>> prec
-        tensor([0.3333, 0.0000, 0.0000, 1.0000])
-        >>> recall
-        tensor([1., 0., 0., 0.])
-        >>> thr
-        tensor([1., 2., 3.])
+        >>> metric(pred, target)   # doctest: +NORMALIZE_WHITESPACE
+        (tensor([0.3333, 0.0000, 0.0000, 1.0000]),
+         tensor([1., 0., 0., 0.]),
+         tensor([1., 2., 3.]))
```

Comment:
I find the previous version more expressive, and easier to read. A simple look at the example already told me how to use the metric, but now it is not clear what the tuple contains, and I have to go look at the forward method documentation.

Reply:
On second thought I agree with you, so I have changed the example back to the original